<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InADayWidget</class>
 <widget class="QMainWindow" name="InADayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>热点</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="MouseSpectrogram" name="mouseSpectrogram" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="KeyboardSpectrogram" name="keyboardSpectrogram" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>操作列表</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>高级</string>
    </property>
    <addaction name="operationListAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>选择程序</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>69</width>
          <height>462</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="ChooseProgramList" name="chooseProgramList" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>操作</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="InADayOperationChart" name="inADayOperationChart" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>时间</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="InADayTimeChart" name="inADayTimeChart" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="operationListAction">
   <property name="text">
    <string>显示操作列表</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChooseProgramList</class>
   <extends>QWidget</extends>
   <header>chooseprogramlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InADayOperationChart</class>
   <extends>QWidget</extends>
   <header>inadayoperationchart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InADayTimeChart</class>
   <extends>QWidget</extends>
   <header>inadaytimechart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MouseSpectrogram</class>
   <extends>QWidget</extends>
   <header>mousespectrogram.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KeyboardSpectrogram</class>
   <extends>QWidget</extends>
   <header>keyboardspectrogram.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
